package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/main.js",
   "scripts": {
     "build": "rm -rf dist && tsc",
-    "codegen": "npm run codegen:graphql-types && npm run codegen:openapi-types",
+    "codegen": "npm run codegen:graphql-types",
```

Comment:
Is a combined job to run both scripts not needed any more? I think it's used by a pre-commit hook

Reply:
Forgot to add an item under 'additional changes':

> pre-commit failed on the OpenAPI validation, so I disabled that. I created [MC-952](https://mozilla-hub.atlassian.net/browse/MC-952) in the maintenance backlog to turn this back on.